CIScripts/VMUtils.ps1

```diff
@@ -64,9 +64,10 @@ function New-TestbedVMs {
 
         $AllDatastores = Get-Datastore -Name $VMCreationSettings.DatastoresList
         $EmptiestDatastore = ($AllDatastores | Sort-Object -Property FreeSpaceGB -Descending)[0]
+        $BaseSnapshot = (Get-Snapshot -VM $Template)[0]
```

Comment:
As we talked - `$BaseSnapshot` could be a parameter. But I personally think this is fine. However, maybe we should add an assert about length of array returned by `Get-Snapshot` to be 1?